### ISSUE TITLE
Errors showing no markdown

### ISSUE THREAD

**Issue by Ryan-Current** [MyNihongo/MudBlazor.Markdown#274]
I ran into an issue with this library where no display will be made at all, without an error. 

The issue I ran into included opening a code block in one cell of a table and closing it in another. This made the table disappear completely, without any errors. 

Here's an example, the changed cell between the tables is the bottom right cell. 

```html
@* Shows a table *@
<MudMarkdown Value="@markdownWorking" />

@* Breaks and shows nothing when the code block is closed *@
<MudMarkdown Value="@markdownNotWorking" />


@code {
    private string markdownWorking = 
    "### My Table: \n\n" +
    "| **Column 1** | **Column 2**   | **Column 3**                |\n" + 
    "|--------------|----------------|-----------------------------|\n" +
    "| Row 1, Col 1 | Row 1, Col 2   |                             |\n" + 
    "|              | Row 2, Col 2   | Row 2, Col 3                |\n" + 
    "| Row 3, Col 1 |                | \u0060\u0060\u0060python    |\n" +
    "|              | Row 4, Col 2   | def greet(name):            |\n" + 
    "| Row 5, Col 1 |                |     return name             |\n" +
    "|              |                |                             |";


    private string markdownNotWorking = 
    "### My Table: \n\n" +
    "| **Column 1** | **Column 2**   | **Column 3**                |\n" + 
    "|--------------|----------------|-----------------------------|\n" +
    "| Row 1, Col 1 | Row 1, Col 2   |                             |\n" + 
    "|              | Row 2, Col 2   | Row 2, Col 3                |\n" + 
    "| Row 3, Col 1 |                | \u0060\u0060\u0060python    |\n" +
    "|              | Row 4, Col 2   | def greet(name):            |\n" + 
    "| Row 5, Col 1 |                |     return name             |\n" +
    "|              |                | \u0060\u0060\u0060          |";
    
}
```

I would prefer to either get an error, or turn off the formatting if an error occurs but still show the original string, rather than a silent fail with no text displayed.

**Comment by MihailsKuzmins** [comment]
Makrdown:
```md
### My Table:

| **Column 1** | **Column 2**   | **Column 3**                |
|--------------|----------------|-----------------------------|
| Row 1, Col 1 | Row 1, Col 2   |                             |
|              | Row 2, Col 2   | Row 2, Col 3                |
| Row 3, Col 1 |                | \u0060\u0060\u0060python    |
|              | Row 4, Col 2   | def greet(name):            |
| Row 5, Col 1 |                |     return name             |
|              |                | \u0060\u0060\u0060          |
```

Expected render output:
![image](https://github.com/user-attachments/assets/0587cd18-24a1-4936-a9df-50641bc23d1d)

**Comment by MihailsKuzmins** [comment]
@Ryan-Current 
In order to make it work I would advise you to use a `@` string as it is shown down below:
```cs
    private string markdownNotWorking =
@"
### My Table:

| **Column 1** | **Column 2**   | **Column 3**                |
|--------------|----------------|-----------------------------|
| Row 1, Col 1 | Row 1, Col 2   |                             |
|              | Row 2, Col 2   | Row 2, Col 3                |
| Row 3, Col 1 |                | \u0060\u0060\u0060python    |
|              | Row 4, Col 2   | def greet(name):            |
| Row 5, Col 1 |                |     return name             |
|              |                | \u0060\u0060\u0060          |
";
```

![image](https://github.com/user-attachments/assets/6442c189-b0b6-451e-83bb-7c922bbfbb1c)

But I understand that you want to be able to see the original markdown if something fails, so after this branch is merged this is what will be rendered:
![image](https://github.com/user-attachments/assets/ef366361-e3fb-49d7-861b-5541c2fd3b1a)

**Comment by Ryan-Current** [comment]
@MihailsKuzmins Thank you for looking into this, and so quickly!

Since this is completed, does that mean a release will be created soon?

**Comment by MihailsKuzmins** [comment]
@Ryan-Current yes, I'm planning to release a new version (including this fix) as soon as a new MudBlazor version is released (probably v7.9.0)
